h2o-py/h2o/estimators/deeplearning.py

```diff
@@ -1000,18 +1000,19 @@ def stopping_rounds(self, stopping_rounds):
     @property
     def stopping_metric(self):
         """
-        Metric to use for early stopping (AUTO: logloss for classification, deviance for regression). Note that custom
-        and custom_increasing can only be used in GBM and DRF with the Python client.
+        Metric to use for early stopping (AUTO: logloss for classification, deviance for regression and anonomaly_score
+        for Isolation Forest). Note that custom and custom_increasing can only be used in GBM and DRF with the Python
+        client.
 
         One of: ``"auto"``, ``"deviance"``, ``"logloss"``, ``"mse"``, ``"rmse"``, ``"mae"``, ``"rmsle"``, ``"auc"``,
-        ``"lift_top_group"``, ``"misclassification"``, ``"mean_per_class_error"``, ``"custom"``, ``"custom_increasing"``
-        (default: ``"auto"``).
+        ``"lift_top_group"``, ``"misclassification"``, ``"mean_per_class_error"``, ``"anomaly_score"``, ``"custom"``,
+        ``"custom_increasing"``  (default: ``"auto"``).
```

Comment:
Can be `anomaly_score` be used for other algos? If not, does it make sense to have `anomaly_score` in other algos as the possibility if a user can't use it?

Reply:
It cannot but this is a current H2O limitation. This list is a union of everything available in H2O (eg. deep learning doesn't support custom metrics either).

Good point though.